### PR TITLE
Added Sponsors Name and Re-organize the docs, images

### LINKED ISSUE #115: Name the sponsor for the first versions in the README
With the logo and link to https://www.hirschmann-stiftung.ch/

### PR DESCRIPTION
**Pull request by HimaTeju**
## 

[//]: # (Briefly explain the reason behind this change.)
Closes #115 

## Changes

[//]: # (Summarize what has changed.)
Added logo, name, and link to the website in readme.
Organized the docs and images into separate folder.

### CODE REVIEW

**Comment by benoit74** [comment]
The fact that the logo is almost white is really weird:

image

@Popolechien are we allowed to modify the logo to remove all the white padding? or is it better to simply not have any logo at all?

**Comment by Popolechien** [comment]
@benoit74 Yes we can remove the padding, we added just so it would fit on the website.

**Review by benoit74 — CHANGES_REQUESTED** [review]
Please:
- use this version: https://skat-foundation.ch/wp-content/uploads/2023/04/logo_hirschmann_stiftung-1.jpg
- add the logo file to the source code instead of directly using the link, so that if for any reason the online file is removed the README is not broken
- simplify the phrase: `Development of this tool was made possible through a grant from [Hirschmann Stiftung](https://www.hirschmann-stiftung.ch/)` (no more bold, no more duplicate reference to the company)

**Comment by HimaTeju** [comment]
@benoit74 I am thinking of gathering all the images used into a separate folder for better organization since currently, the images are scattered. What's your opinion on this?

**Comment by benoit74** [comment]
> I am thinking of gathering all the images used into a separate folder for better organization since currently, the images are scattered. What's your opinion on this?

I agree there is maybe already too much things on the root folder. Maybe move all markdown (except README.md and CHANGELOG.md) to a `docs` folder + all images to a `docs/images` folder ? This means the logo will be a bit far away from the README.md but we can live with it.

**Comment by benoit74** [comment]
Is this now ready for new review?

**Comment by HimaTeju** [comment]
@benoit74 I have made the necessary changes as you said. I have also used the new relative paths of the docs and images but the readme file is broken. It is not redirecting to the required docs/images.

**Comment by HimaTeju** [comment]
@benoit74 PR is ready for the new review!

File: docs/architecture_data.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Metrics - Data architecture
 =======
 
-![Data architecture](architecture_data.excalidraw.png)
+![Data architecture](docs/images/architecture_data.excalidraw.png)
```
Comment (benoit74) [inline docs/architecture_data.md:4]: Nope, this leads to `docs/docs/images/architecture_data.excalidraw.png` which does not exists. We are already in the `docs` subfolder so should be `images/architecture_data.excalidraw.png`

File: docs/architecture_technical.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ At some point in the future, the system is meant to centralize data in a Cloud f
 
 ## High-level overview
 
-![Technical architecture](architecture_technical.excalidraw.png)
+![Technical architecture](docs/images/architecture_technical.excalidraw.png)
```
Comment (benoit74) [inline docs/architecture_technical.md:12]: Idem

File: docs/database.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ One indicator record hold the value of:
 - for a given [period](#indicator-periods) (8. July 2023 at 8am)
 - and a given [dimension](#indicator-dimensions) (wikipedia)
 
-![Indicators record](database_indicator_record.excalidraw.png)
+![Indicators record](docs/images/database_indicator_record.excalidraw.png)
```
Comment (benoit74) [inline docs/database.md:11]: Idem

File: docs/database.md [hunk 4]
```diff
@@ -38,7 +38,7 @@ Indicator dimensions might need multiple values for one dimension. For instance
 
 Indicator states are the transient values of the indicators before they are transformed in records every hour.
 
-![Indicators state](database_indicator_state.excalidraw.png)
+![Indicators state](docs/images/database_indicator_state.excalidraw.png)
```
Comment (benoit74) [inline docs/database.md:41]: Idem

File: docs/database.md [hunk 5]
```diff
@@ -56,6 +56,6 @@ One single table is used to store KPIs with following columns:
 - `agg_value`: the value of the aggregation for this KPI value (e.g 2023-06-01 for a daily aggregation, 2023 for a yearly aggregation, ...)
 - `kpi_value`: the value of the given KPI for the given `agg_kind` and `agg_value` (e.g. the value of KPI id `1` for the `Month` of `March 2023`)
 
-![KPI](database_kpi.excalidraw.png)
+![KPI](docs/images/database_kpi.excalidraw.png)
```
Comment (benoit74) [inline docs/database.md:59]: Idem

**Review by benoit74 — CHANGES_REQUESTED** [review]
Still not OK, it looks like you did not tested your modifications that much (quite straightforward to see the issues mentioned, just opening https://github.com/HimaTeju/metrics/blob/Add-Sponsor-to-readme/docs/architecture_data.md shows that image is broken)

File: docs/database.md (hunk 5 above)
Comment (HimaTeju) [reply docs/database.md]: Shall I rename this PR? This PR is doing more than just "adding sponsors name"

**Review by benoit74 — APPROVED** [review]
LGTM

Please update PR title and first comment as you suggested (feel free to do without asking next time, PR title and first comment should represent the final content of the PR as much as possible). 

I will merge anyway, you can still edit the PR title and first comment after the merge.

**Pull request merged**